### DISCUSSION TITLE
What are common operations that can be performed on a data-structure?

### DISCUSSION THREAD

**Discussion by ankit-akash** [ankit-akash/DSA#35] [Q&A] [answered]
All the possible operations

**Comment by fcasamento** [comment, marked as answer]
I think there must be many more, but I only remember these here:

1. Creation
2. Insertion
3. Traversal
4. Search
5. Sorting
6. Merging
7. Updation
8. Deletion
9. Array
10. Stack
11. List
12. Queue
13. Tree
14. Graph
15. Split Data

**Reply by fcasamento** [reply]
@ankit-akash Mark this as an **answer** as well, we can help each other. 🙏🏻 Pls
